Makefile

```diff
@@ -38,3 +41,6 @@ linux:
 osx:
 	$(CXX) -m64 $(CXXFLAGS) -o ./ext-libs/pbf2geojson/pbf2geojson_osx_x86_64.so ./ext-libs/pbf2geojson/pbf2geojson.cpp
 	$(CXX) -m32 $(CXXFLAGS) -o ./ext-libs/pbf2geojson/pbf2geojson_osx_i686.so ./ext-libs/pbf2geojson/pbf2geojson.cpp
+
+openbsd:
+	$(CXX) -fPIC -std=c++11 -I/usr/local/include -s -Os -Ofast -O3 --shared -o ./ext-libs/pbf2geojson/pbf2geojson_openbsd_x86_64.so ./ext-libs/pbf2geojson/pbf2geojson.cpp
```

Comment:
why not use $(CXXFLAGS) and add -I/usr/local/include in CXXFLAGS?

Reply:
```
CXXFLAGS = -std=c++11 -Iinclude -I../protozero/include -I../vtzero/include -I/usr/local/include -s -Os -Ofast -O3 --shared
```

Reply:
For FreeBSD:

```
freebsd:                                                                                                                                                                                                                                                               
    $(CXX) -m64 $(CXXFLAGS) -I/usr/local/include -o ./ext-libs/pbf2geojson/pbf2geojson_freebsd_x86_64.so ./ext-libs/pbf2geojson/pbf2geojson.cpp
    $(CXX) -m32 $(CXXFLAGS) -I/usr/local/include -o ./ext-libs/pbf2geojson/pbf2geojson_freebsd_i686.so ./ext-libs/pbf2geojson/pbf2geojson.cpp
```

Reply:
right, reused CXXFLAGS and added the FreeBSD bits